### PR TITLE
DCS-1494 email to prison contains court email address for new bookings

### PR DESCRIPTION
**Pull request by GurnankCheema**
Court email address will only be displayed in emails sent to prisons.

### CODE REVIEW

File: backend/config.js [hunk 1]
```diff
@@ -74,7 +74,9 @@ module.exports = {
     notifyKey: process.env.NOTIFY_API_KEY || '',
 
     bookingCreationConfirmationCourt: '47cfc1c3-ad56-47c4-98e7-0e41c0e2c9b4',
-    bookingCreationConfirmationPrison: '05456200-846b-446e-81cb-e003611bfea6',
+
+    bookingCreationConfirmationPrison: '7a5fe332-0001-4ef2-bd06-5595c01552f5',
+    bookingCreationConfirmationPrisonWithCourtEmailAddress: 'f659c351-da69-4a59-984b-8345e59c6048',
```
Comment (GurnankCheema) [inline backend/config.js:79]: See Notify, folder March 2022 prisonNotifications

File: backend/services/notificationService.test.ts [hunk 2]
```diff
@@ -616,7 +619,7 @@ describe('Notification service', () => {
       await notificationService.sendBookingCreationEmails(context, 'A_USER', createEmail)
 
       expect(notifyApi.sendEmail).toHaveBeenCalledWith(
-        config.notifications.bookingCreationConfirmationPrison,
+        config.notifications.bookingCreationConfirmationPrisonWithCourtEmailAddress,
```
Comment (andrewrlee) [inline backend/services/notificationService.test.ts:622]: will need to add a test to cover the other situation

Comment (GurnankCheema) [reply backend/services/notificationService.test.ts:622]: added

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**